lib/Views/PopupMessage.html

```diff
@@ -1,7 +1,7 @@
 <div class="modal-background" data-bind="click: closeIfClickOnBackground">
     <div class="popup-message" data-bind="style: { width: width + 'px', height: height + 'px', 'max-width': maxWidth + 'px', 'max-height': maxHeight + 'px' }">
         <div class="modal-header">
-            <div class="modal-close-button" data-bind="click: close">&times;</div>
+            <div class="modal-close-button" data-bind="click: close, hasFocus: true">&times;</div>
```

Comment:
I don't think this is working.  I can't seem to control a modal dialog with keyboard controls.  To test this, open the Chrome debugger, Resources tab, Local Storage, localhost, and delete the `NationalMap.AbsDataProviderWarning` key.  Then open the `Australian Bureau of Statistics (BETA)` folder.